<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="shutButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>关机</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabwidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>371</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>系统信息</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>主机名:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>启动时间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>运行时间:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>系统版本号:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU型号与主频:</string>
      </property>
     </widget>
     <widget class="QLabel" name="hostname">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>xxxxx</string>
      </property>
     </widget>
     <widget class="QLabel" name="start_time">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>xxxxxxx</string>
      </property>
     </widget>
     <widget class="QLabel" name="run_time">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>xxxxxxxxx</string>
      </property>
     </widget>
     <widget class="QLabel" name="system_version">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>xxxxxxxxx</string>
      </property>
     </widget>
     <widget class="QLabel" name="CPU_model">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>421</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>xxxxxxxxx</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>进程信息</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>561</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>操作</string>
     </attribute>
     <widget class="QLabel" name="pid_input">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="proc_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>进程名:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="query1">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="query2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="killproc">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>结束进程</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>541</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab4">
     <attribute name="title">
      <string>记录曲线</string>
     </attribute>
     <widget class="QLabel" name="CPU_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>511</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>CPU使用率曲线</string>
      </property>
     </widget>
     <widget class="QLabel" name="Mem_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>511</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>内存使用率曲线</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="run">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>运行新任务</string>
    </property>
   </widget>
   <zorder>tabwidget</zorder>
   <zorder>shutButton</zorder>
   <zorder>run</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
